test/response_test.rb

```diff
@@ -972,6 +972,16 @@ def generate_audience_error(expected, actual)
         assert_empty response_valid_signed.errors
       end
 
+      it "return true when at least a cert on idp_cert_multi is valid and keys are strings" do
+        settings.idp_cert_multi = {
+          "signing" => [ruby_saml_cert_text2, ruby_saml_cert_text],
+          "encryption" => []
+        }
+        response_valid_signed.settings = settings
+        res = response_valid_signed.send(:validate_signature)
```

Comment:
In addition to this unittest,  I expect one using a call to get_idp_cert_multi as well that verifies the certs were properly loaded on the settings object at settings_test.rb

Reply:
Added tests verifying that calls to get_idp_cert_multi are in fact working: 01c95db